dagshub/data_engine/client/data_client.py

```diff
@@ -165,6 +220,9 @@ def delete_datasource(self, datasource: Datasource):
         return self._exec(q, params)
 
     def scan_datasource(self, datasource: Datasource):
+        """
+        Initiate a scan operation on the specified datasource.
```

Comment:
Doesn't this need the argument?

Also, what does this scan? This should be in the description

Reply:
I think it returns the parameters of a datasource. Will update it after confirming.